Source/Core/Gen5/Generators/IDGenerator5.hpp

```diff
@@ -28,7 +28,7 @@ class IDGenerator5 : public IDGenerator
 public:
     IDGenerator5() = default;
     IDGenerator5(u32 initialAdvances, u32 maxAdvances, const IDFilter &filter);
-    std::vector<IDState5> generate(u64 seed, u32 pid = 0, bool checkPID = false);
+    std::vector<IDState5> generate(u64 seed, u32 pid = 0, bool checkPID = false, bool checkXOR = false);
```

Comment:
I know part of this is me, but neither of these need a default value. Please remove it from both checkPID and checkXOR.

Reply:
Default values have been removed